packages/aws-client/src/S3.ts

```diff
@@ -246,6 +247,21 @@ export const s3GetObjectTagging = improveStackTrace(
     s3().getObjectTagging({ Bucket: bucket, Key: key }).promise()
 );
 
+const getObjectTags = async (bucket: string, key: string) => {
+  const taggingResponse = await s3GetObjectTagging(bucket, key);
+
+  return taggingResponse.TagSet.reduce(
+    (acc, { Key, Value }) => ({ ...acc, [Key]: Value }),
+    {}
+  );
+};
+
+const getObjectTaggingString = async (bucket: string, key: string) => {
+  const tags = await getObjectTags(bucket, key);
+
+  return querystring.stringify(tags);
+};
```

Comment:
My vote - either is probably fine. 

As a matter of personal preference I tend to prefer explicit/readable over terse when there's not a compelling reason to do so.    In this case I feel like I have to do a short bit of mental unwinding to the later to identify what I'm getting back when casually reading it, *probably* because we're generally not using promise chains elsewise.   

 It's admittedly a *short* number of cycles, but enough to tip the preference scale to my mind.

Reply:
Agreed. I tend to use `async/await` vs Promise chains because I think that `async/await` makes it easier on the reader, whether they're a new-to-JS dev or a tired senior dev.

Reply:
Cool. Appreciate the feedback. Absolutely agree on readability over terseness, in general.

In this case, it seems perhaps to be something that tips towards whichever convention you're used to. I imagine if the common case were reversed (i.e., the way I wrote it instead), your response would be in favor of the reverse (since it would be the common case).